espflash/src/cli/monitor/mod.rs

```diff
@@ -33,6 +34,17 @@ pub mod parser;
 mod line_endings;
 mod symbols;
 
+#[cfg_attr(feature = "cli", derive(clap::ValueEnum))]
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Display, EnumIter, EnumString, EnumVariantNames)]
+#[non_exhaustive]
+#[strum(serialize_all = "lowercase")]
+pub enum LogFormat {
+    /// defmt
+    Defmt,
+    /// serial
+    Serial,
```

Comment:
We should make `Serial` the default option, not sure the best way to do this in clap so I'll leave it to you :D.

Reply:
^ I wrote this same comment and forgot to send it haha.

Reply:
This should now be fixed with https://github.com/esp-rs/espflash/pull/493/commits/5273a4a8f1d14b6849029ea631f39115bb4f3ce4